components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -90,6 +92,8 @@ abstract class AbstractFetchDownloadService : Service() {
 
     protected abstract val httpClient: Client
 
+    protected abstract val style: Style
```

Comment:
instead of making `style` abstract and force consumers to provide a value, we could make it [open](https://kotlinlang.org/docs/reference/classes.html#inheritance) and provide a default `style` and consumer that want to modify it will have to option to do so. What do you think? :)